src/main/java/org/qiot/covid19/datahub/registration/rest/RegisterResource.java

```diff
@@ -3,65 +3,85 @@
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 import javax.transaction.Transactional;
-import javax.validation.constraints.NotNull;
 import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
 import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
-import org.jboss.resteasy.plugins.providers.multipart.MultipartFormDataOutput;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.resteasy.spi.NotImplementedYetException;
+import org.qiot.covid19.datahub.registration.client.StationServiceClient;
+import org.qiot.covid19.datahub.registration.rest.beans.RegisterRequest;
+import org.qiot.covid19.datahub.registration.rest.beans.RegisterResponse;
 import org.qiot.covid19.datahub.registration.service.CertificateService;
-import org.qiot.covid19.datahub.registration.service.RegistrationService;
+import org.slf4j.Logger;
 
 @Path("/register")
 @ApplicationScoped
-@Produces(MediaType.TEXT_PLAIN)
-@Consumes(MediaType.TEXT_PLAIN)
 public class RegisterResource {
 
     @Inject
-    RegistrationService registrationService;
+    @RestClient
+    StationServiceClient stationServiceClient;
     @Inject
     CertificateService certificateService;
 
+    @Inject
+    Logger LOGGER;
+
+    /**
+     * Creates a new instance of a `RegisterRequest`.
+     */
     @Transactional
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response register(RegisterRequest data) throws Exception {
+        LOGGER.debug("Received registerRequest: {}", data);
```

Comment:
change LOGGER.debug to info ... it is always good to know the call behaviour ...